backend/ops_api/ops/schemas/budget_line_items.py

```diff
@@ -269,6 +269,15 @@ class Meta:
     email = fields.Str(default=None, allow_none=True)
 
 
+class PortfolioTeamLeadersSchema(Schema):
+    class Meta:
+        unknown = EXCLUDE  # Exclude unknown fields
+
+    id = fields.Int(required=True)
+    full_name = fields.Str(default=None, allow_none=True)
```

Comment:
instead of default=None, would recommend using `load_default` (for de-serialization) or `dump_default` (for serialization).   default is being [deprecated in current versions of marshmallow](https://marshmallow.readthedocs.io/en/3.x-line/marshmallow.fields.html#). 

```Changed in version 3.13.0: Replace missing and default parameters with load_default and dump_default.```